### DISCUSSION TITLE
Using addon in MDX & TSX files

### DISCUSSION THREAD

**Discussion by pattimcletchie** [storybookjs/addon-designs#155] [Q&A] [answered]
I am trying to just use the "figma block" like, https://pocka.github.io/storybook-addon-designs/?path=/docs/docs-advanced-usage--embed-multiple-designs#doc-blocks this example but when I use this import `import { Figma } from "storybook-addon-designs/blocks";` I get a type error in my `.stories.tsx` page

Type error: `Cannot find module 'storybook-addon-designs/blocks' or its corresponding type declarations.` 

Any thoughts on how best to use the Figma block when not using story decorator params?

We are using Storybook v6.5.8 and  "storybook-addon-designs": "^6.2.1"

**Comment by pocka** [comment, marked as answer]
@pattimcletchie Sorry, I didn't consider TS usage for that file. For a workaround, you can import from `storybook-addon-designs/esm/blocks` or `storybook-addon-designs/cjs/blocks`.

#156

**Reply by pattimcletchie** [reply]
Thank you so much! That worked perfectly.

**Reply by pocka** [reply]
I published the new version `storybook-addon-designs@6.3.1`, which adds `blocks.d.ts` type definition file, so you can import from `storybook-addon-designs/blocks` without workarounds. If you still encounter type errors, please let me know.
